src/x509_scitokens_issuer/x509_scitokens_issuer.py

```diff
@@ -19,22 +19,31 @@
 from flask import Flask, request
 
 # Load the application and configuration defaults.
+ipath = '/usr/share/x509-scitokens-issuer'
 if platform.system() == 'Darwin':
-    app = Flask(__name__, instance_path="/usr/local/share/x509-scitokens-issuer", instance_relative_config=True)
-else:
-    app = Flask(__name__, instance_path="/usr/share/x509-scitokens-issuer", instance_relative_config=True)
+    ipath = '/usr/local/share/x509-scitokens-issuer'
+instance_path = os.environ.get('X509_SCITOKENS_ISSUER_INSTANCE_PATH', ipath)
+app = Flask(__name__, instance_path=instance_path, instance_relative_config=True)
 app.updater_thread = None
 app.issuer_key = None
 
 def _load_default_config():
-    app.config.update({
+    aconf = os.environ.get('X509_SCITOKENS_ISSUER_CONFIG', '')
+    if aconf:
+        print("Loading {} config".format(aconf))
+        conf = json.load(open(aconf))
+    else:
+        print("Loading default config")
+        conf = {
```

Comment:
I did not change the logic of the code. By default it will work as before, I only provided possibility to supply custom config.

Reply:
Right - I'm just double-checking that this is what you intend.

This will remove all default variable values when you supply the environment variable, meaning you're on the hook to specify all of them (even the ones like `LIFETIME` that don't involve paths).

Reply:
Ok, I modify it accordingly to perform partial update then. This will keep the defaults if external config will not supply them.